nginx/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM nginx:1.26.1
+
+COPY conf/nginx.conf /etc/nginx/conf.d/default.conf
+
```

Comment:
Missing expose and volume declaration for documentation

Reply:
ok